cmake/kokkos_enable_devices.cmake

```diff
@@ -88,3 +88,5 @@ ENDIF()
 KOKKOS_DEVICE_OPTION(SERIAL ${SERIAL_DEFAULT} HOST "Whether to build serial backend")
 
 KOKKOS_DEVICE_OPTION(HPX OFF HOST "Whether to build HPX backend (experimental)")
+
+KOKKOS_DEVICE_OPTION(HIP OFF DEVICE "Whether to build HIP backend")
```

Comment:
We are sure we don't need flags like the OpenMP thing here?

Reply:
I was able to build just by setting the cxx compiler to `hipcc`.  Bruno is working on a different system and had to set `--amdgpu-target`.

Reply:
yeah I got this in the Makefile build:
```
clang-10: warning: argument unused during compilation: '--amdgpu-target=gfx900' [-Wunused-command-line-argument]
```